hive-bigquery-parent/pom.xml

```diff
@@ -62,7 +62,7 @@
         <jackson.version>2.12.7.1</jackson.version>
         <protobuf.version>3.21.12</protobuf.version>
         <legacy-bigquery-connector.version>hadoop2-1.0.0</legacy-bigquery-connector.version>
-        <bigquery-connector-common.version>0.30.0</bigquery-connector-common.version>
+        <bigquery-connector-common.version>0.0.1-SNAPSHOT</bigquery-connector-common.version>
```

Comment:
Please convert to version 0.31.0 (soon to be released)

Reply:
Done